### ISSUE TITLE
mozjpeg fails during npm install on OSX

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/zoo-reduxify#13]
```
> mozjpeg@4.1.1 postinstall /Users/odonnell/zooniverse/pandora/node_modules/mozjpeg
> node lib/install.js

  ⚠ The `/Users/odonnell/zooniverse/pandora/node_modules/mozjpeg/vendor/cjpeg` binary doesn't seem to work correctly
  ⚠ mozjpeg pre-build test failed
  ℹ compiling from source
  ✖ Error: autoreconf -fiv && ./configure --disable-shared --prefix="/Users/odonnell/zooniverse/pandora/node_modules/mozjpeg/vendor" --bindir="/Users/odonnell/zooniverse/pandora/node_modules/mozjpeg/vendor" --libdir="/Users/odonnell/zooniverse/pandora/node_modules/mozjpeg/vendor" && make --jobs=4 && make install --jobs=4
Command failed: ./configure --disable-shared --prefix="/Users/odonnell/zooniverse/pandora/node_modules/mozjpeg/vendor" --bindir="/Users/odonnell/zooniverse/pandora/node_modules/mozjpeg/vendor" --libdir="/Users/odonnell/zooniverse/pandora/node_modules/mozjpeg/vendor"
configure: error: installation or configuration problem: assembler cannot create object files.

    at ChildProcess.exithandler (child_process.js:204:12)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:886:16)
    at Socket.<anonymous> (internal/child_process.js:342:11)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at Pipe._handle.close [as _onclose] (net.js:501:12)
```

**Comment by eatyourgreens** [comment]
Doesn't seem to actually break anything, and I can't see which dependency is using it.

**Comment by simoneduca** [comment]
[After installing `nasm`](https://github.com/imagemin/mozjpeg-bin/issues/6), the pre-build test still fails but the build is successful.
![screen shot 2017-05-25 at 15 46 07](https://cloud.githubusercontent.com/assets/8989606/26455320/c3381838-4161-11e7-8ace-bb8b5c4bca2a.png)

**Comment by srallen** [comment]
I'm not sure if this is fixed with #15, but I didn't see this happening with all the updated dependencies.

**Comment by srallen** [comment]
This might be fixed in #17 by adding 
```
node: {
    fs: 'empty'
}
```
to the webpack configs.
